public/js/Bullet.js

```diff
@@ -65,7 +68,11 @@ class Bullet extends GameObject {
   draw(ctx: Object) {
     ctx.fillStyle = this._color;
     ctx.beginPath();
-    ctx.arc(this._x, this._y, this._size, 0, 2*Math.PI);
+    ctx.arc(
+      this._x * Globals.widthRatio,
+      this._y * Globals.heightRatio,
+      this._size * Globals.widthRatio,
+      0, 2*Math.PI);
```

Comment:
Also add a ',' after this line, and do this for every time you move the ')' down